apidoc/ostree-sections.txt

```diff
@@ -618,17 +618,6 @@ ostree_remote_get_url
 ostree_remote_get_type
 </SECTION>
 
-<SECTION>
-<FILE>ostree-repo-experimental</FILE>
-ostree_repo_find_remotes_async
-ostree_repo_find_remotes_finish
-ostree_repo_pull_from_remotes_async
-ostree_repo_pull_from_remotes_finish
-ostree_repo_resolve_keyring_for_collection
-OSTREE_REPO_METADATA_REF
-OSTREE_META_KEY_DEPLOY_COLLECTION_ID
-</SECTION>
```

Comment:
I think this is and the hunk below are unintended. They are no longer experimental after 8fbf19c9f54ba34ae7ebbc540b08566b8dd18848 and I think this was just not clearly renaming the section when shifting it from `apidoc/ostree-experimental-sections.txt` in that commit. Probably these should be moved into the regular `ostree-repo` section.